Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -517,6 +517,7 @@ private void OnKeyDown(object sender, KeyEventArgs e)
                     if (specialKeyState.CtrlPressed)
                     {
                         if (_viewModel.SelectedIsFromQueryResults()
+                            && QueryTextBox.Text.Length > 0
```

Comment:
I think we can use a $?$ operator lower to resolve this as well. But this shall work as well.